src/components/MultiHopTrade/components/LimitOrderV2/LimitOrderConfirm.tsx

```diff
@@ -96,9 +96,10 @@ export const LimitOrderConfirm = () => {
     approvalNetworkFeeCryptoBaseUnit,
   } = useAllowanceApproval({
     activeQuote,
-    isQueryEnabled:
-      !!allowanceApproval.isInitiallyRequired &&
-      !!activeQuote &&
+    isQueryEnabled: !!allowanceApproval.isInitiallyRequired && !!activeQuote,
+    // Stop refetching when the approval tx is pending, but keep it enabled (see above) so we can leverage stale data, see
+    // https://github.com/shapeshift/web/issues/8702
+    isRefetchEnabled:
```

Comment:
This and the below are the guts of this PR, everything else is pretty much just plumbing.